peribolos/knative.yaml

```diff
@@ -152,7 +152,6 @@ orgs:
     - yuvalif
     - yuzisun
     - zainabhusain227
-    - Zerpet
```

Comment:
This line will effectively remove you from `Knative` gh org, if that's also desired. Then, you want to do the same on knative-extensions.yaml. But that's solely on your decision AFAIK.

Reply:
Yeah, this is ok. I don't think I'll get time to contribute to this community any time soon 😔